src/index.js

```diff
@@ -15,6 +15,18 @@ const etherConfig = {
     [process.env.REACT_APP_ETHER_CHAIN_ID]: process.env.REACT_APP_ETHER_PROVIDER,
   },
 };
+const adminAuthConfig = {
+  clientId: `${process.env.REACT_APP_SSO_API_ADMIN_CLIENT_ID}`,
+  authorizationEndpoint: `${process.env.REACT_APP_SSO_API}/oauth/authorize`,
+  tokenEndpoint: `${process.env.REACT_APP_SSO_API}/oauth/token`,
+  redirectUri: `${process.env.REACT_APP_FRONTEND_REDIRECT}/admin=true`,
```

Comment:
This url was created in the same way in few files we should avoid that an use some function that would create it instead. Thanks to that in future if we will have any update to "creation" of this url we simple change it in one file instead of searching it in whole project. (I believe the same approach would be nice do be done for the rest of does links as well)

Reply:
I agree with the admin link, this is a special case that needs to be handled, for the other ones, its ok as is, because this is the only place we should ever have these links in.